HelpAssistHer/help-assist-her
# Geocode address

## Description
This PR actually geocodes the address that's typed into the address input and displays results within 50 miles of that address!

## Test Plan
1. Go to http://localhost:4000/mini-app
1. Type in "new york, ny"
1. See that you get a list of pregnancy centers
1. Go back to http://localhost:4000/mini-app
1. Type in "new jersey"
1. See that you get a different list of results
